scopes/design/ui/Avatar/user-avatar.tsx

```diff
@@ -25,16 +43,20 @@ export class UserAvatar extends PureComponent<UserAvatarProps> {
       fontSize = Math.round(size * 0.4),
       className,
       imgClassName,
+      hideTooltip = true,
       ...rest
     } = this.props;
     const { profileImage = '', name = '', displayName = '' } = account;
     const firstLetter = name[0] || displayName[0];
     const profileImageWithParams = addAvatarQueryParams(profileImage, imageSize, styles.defaultAvatarBgColor);
     const colors = firstLetter && letterBgColors[firstLetter.toLowerCase()];
+    const tooltipId = v1();
```

Comment:
this is not good, you are recreating tooltipId on every render.
Also, it will break server-side rendering (DOM will be different between server and client)

so,
1. move tooltipId to state (react 15 style)
2. set tooltip onComponentDidMount()
3. render `<Tooltip` when tooltipId is defined, ie.  
```
{!hideTooltip && !!tooltipId && (
  <Tooltip ....
)}
```

Reply:
I made the changes.